powerdns/migrations/0006_auto_20150612_0756.py

```diff
@@ -20,7 +20,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='record',
             name='auto_ptr',
-            field=models.BooleanField(verbose_name='Auto PTR field', default=True),
+            field=models.IntegerField(verbose_name='Auto PTR field', default=True),
```

Comment:
yes but we're changing field type, so default value should match do the field type.